es/build.gradle

```diff
@@ -13,20 +13,12 @@ subprojects {
     }
 
     compileJava {
-        dependsOn rootProject.jdks.runtime
```

Comment:
It is not needed. The compileJava and test tasks were configured in the root build file.